src/pages/what-is-ebpf.js

```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+	    <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+	    there was a vote taken and the bee was named **eBee**.
```

Comment:
Oh and sorry I'm writing too much Markdown :facepalm: 

```suggestion
	    there was a vote taken and the bee was named <b>eBee</b>.
```